Index.html

```diff
@@ -0,0 +1,128 @@
+            <div class="mySkills">
+                <a id="languages" href="#">Languages</a>
+                <ul class="abilities">
+                    <li>
+                        <a href="#">JavaScript</a>
+                    </li>
+                    <li>
+                        <a href="#">Ruby</a>
+                    </li>
+                    <li>
+                        <a href="#">Html</a>
+                    </li>
+                    <li>
+                        <a href="#">Html</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="mySkills">
+                <a id="frameworks" href="#">Frameworks</a>
+                <ul class="abilities">
+                    <li>
+                        <a href="#">Bootstrap</a>
+                    </li>
+                    <li>
+                        <a href="#">Ruby on Rails</a>
+                    </li>
+                    <li>
+                        <a href="#">RSPec</a>
+                    </li>
+                    <li>
+                        <a href="#">CapyBara</a>
+                    </li>
+                    <li>
+                        <a href="#">Selenium</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="mySkills">
+                <a id="skills" href="#">Skills</a>
+                <ul class="abilities">
+                    <li>
+                        <a href="#">Codekit</a>
+                    </li>
+                    <li>
+                        <a href="#">Github</a>
+                    </li>
+                    <li>
+                        <a href="#">Codepen</a>
+                    </li>
+                    <li>
+                        <a href="#">Gitlab</a>
+                    </li>
+                    <li>
+                        <a href="#">Terminal</a>
+                    </li>
+                </ul>
+            </div>
+        <img class="line" src="/assets/Vector 4.svg" alt="horizontal line">
+        </div>
+    </section>
+    <section class="socials">
+        <ul class="profiles">
+            <li>
+                <a href="#">
+                    <img src="/assets/github.svg" alt="GitHub Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img id="linkedIn" src="/assets/linkedinn.png" alt="LinkedIn Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="/assets/angellist.svg" alt="Angellist Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="/assets/Twitter.svg" alt="Twitter Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="/assets/medium.svg" alt="Medium Logo">
+                </a>
+            </li>
+        </ul>
+    </section>
```

Comment:
- [ ] It's good you added these lines of code. But you did go the extra mile than stated for this project.
You are only required to work on the toolbar and the headline section. Let's try to remove these lines of code.